api/errors.js

```diff
@@ -0,0 +1,8 @@
+exports.handleBadPath = (req, res, next) => {
+	res.status(400).send({ msg: "Bad Request" });
+};
```

Comment:
this status code to handle an incorrect path should be a 404 👀

Reply:
Will do, thank you. I've been putting 400 for this all the time haha